pkg/connectorrunner/runner.go

```diff
@@ -584,6 +599,8 @@ func NewConnectorRunner(ctx context.Context, c types.ConnectorServer, opts ...Op
 			tm = local.NewListTicketSchema(ctx)
 		case cfg.getTicketConfig != nil:
 			tm = local.NewGetTicket(ctx, cfg.getTicketConfig.ticketID)
+		case cfg.bulkCreateTicketConfig != nil:
```

Comment:
I think we had talked about maybe just making it so that ticketconfig could have multiple. It's not used much. Not a big deal though.